.eslintrc.yml

```
@@ -0,0 +1,18 @@
+env:
+  browser: true
+  es2021: true
+extends:
+  - eslint:recommended
+  - plugin:react/recommended
+  - plugin:json/recommended
+parserOptions:
+  ecmaFeatures:
+    jsx: true
+  ecmaVersion: 13
+  sourceType: module
+plugins:
+  - react
+rules: {
+  'space-before-function-paren': ['error', 'never'],
+    'comma-dangle': ['error', 'always-multiline'],
```

```suggestion
  'space-before-function-paren': ['error', 'never'],
  'comma-dangle': ['error', 'always-multiline'],
```
These should have the same indent level, I believe.

---

Ok I will do the requested changes